frontend/src/concepts/pipelines/NoPipelineServer.tsx

```diff
@@ -20,7 +13,6 @@ const NoPipelineServer: React.FC<NoPipelineServerProps> = ({ variant }) => (
     <Title headingLevel="h2" size="lg">
       No pipeline server
     </Title>
-    <EmptyStateBody>To import a pipeline, first create a pipeline server.</EmptyStateBody>
```

Comment:
This will (I'm pretty sure) impact the DS Pipeline & Run views as well -- can you take screenshots of those as well and add it to your description?

Reply:
@ppadti I don't think the intent is to affect the other pages. Only the project details page. @andrewballantyne please confirm.

The pipelines and runs pages are affected (see screen shots).

Reply:
The goal was to address all the various states in the Project Details View -- but Pipelines shares items. I am not really against us reaching out to those, but we should be aware of context of why we are changing this.

It was redundant for each section to say "I'm a section, click my button!" over and over. That's what really we are addressing here. I think it's safe to say we can do only that, but we shouldn't just "apply that" to everything.

I have not been able to test this to see what impact is happening here -- the screenshots in the description to look barren on the other pages, I do not recall what was there.

Reply:
Other global pages such as Model Serving and Notebook image settings still have description text. 
The request was for the project details sections, so I suggest the change only affects that area unless UX makes suggests otherwise.

Reply:
@christianvogt the required changes have been addressed and this PR is ready for review, if no further changes required from Model Serving and Notebook image settings page.

Reply:
@ppadti please update your screenshots for the global Pipeline pages

Reply:
@andrewballantyne Sorry I missed that. I have updated the screenshots of global pipeline page as well as DS project page after installing the server.